src/devTool.tsx

```diff
@@ -25,6 +26,13 @@ export const DevTool = <T extends FieldValues>(props?: {
 }) => {
   const methods = useFormContext();
 
+  const { isExtensionEnabled } = useExportControlToExtension(
+    (props && props.control) || methods.control,
```

Comment:
```suggestion
    props?.control ?? methods.control,
```

Reply:
Also Line 39?

Reply:
again no biggy, just minor I have already approved, happy to merge as it is.